packages/react-storage/src/components/StorageBrowser/views/LocationActionView/utils.ts

```diff
@@ -159,3 +162,11 @@ export const getActionViewTableData = <T extends ActionData>({
 
   return { headers: DELETE_ACTION_VIEW_HEADERS, rows };
 };
+
+export const GetTitle = (): string | undefined => {
+  const [{ actionType }] = useStore();
+  const actions = useTempActions();
+  const action = actionType ? actions[actionType] : undefined;
+
+  return action?.options?.displayName;
+};
```

Comment:
After seeing this function... this logic should be inside the view hooks rather than be inside its own function

Reply:
Got it, I'll move it inside each hook